lib/ansible/modules/network/fortimanager/fmgr_secprof_ssl_ssh.py

```diff
@@ -0,0 +1,1095 @@
+      - choice | disable | Disable SSH tunnel policy check.
+      - choice | enable | Enable SSH tunnel policy check.
+    required: false
+    choices: ["disable", "enable"]
+
+  ssh_status:
+    description:
+      - Configure protocol inspection status.
+      - choice | disable | Disable.
+      - choice | deep-inspection | Full SSL inspection.
+    required: false
+    choices: ["disable", "deep-inspection"]
+
+  ssh_unsupported_version:
+    description:
+      - Action based on SSH version being unsupported.
+      - choice | block | Block.
+      - choice | bypass | Bypass.
+    required: false
+    choices: ["block", "bypass"]
+
+  ssl:
+    description:
+      - EXPERTS ONLY! KNOWLEDGE OF FMGR JSON API IS REQUIRED!
+      - List of multiple child objects to be added. Expects a list of dictionaries.
+      - Dictionaries must use FortiManager API parameters, not the ansible ones listed below.
+      - If submitted, all other prefixed sub-parameters ARE IGNORED.
+      - This object is MUTUALLY EXCLUSIVE with its options.
+      - We expect that you know what you are doing with these list parameters, and are leveraging the JSON API Guide.
+      - WHEN IN DOUBT, USE THE SUB OPTIONS BELOW INSTEAD TO CREATE OBJECTS WITH MULTIPLE TASKS
+    required: false
+
+  ssl_allow_invalid_server_cert:
+    description:
+      - When enabled, allows SSL sessions whose server certificate validation failed.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  ssl_client_cert_request:
+    description:
+      - Action based on client certificate request failure.
+      - choice | bypass | Bypass.
+      - choice | inspect | Inspect.
+      - choice | block | Block.
+    required: false
+    choices: ["bypass", "inspect", "block"]
+
+  ssl_inspect_all:
+    description:
+      - Level of SSL inspection.
+      - choice | disable | Disable.
+      - choice | certificate-inspection | Inspect SSL handshake only.
+      - choice | deep-inspection | Full SSL inspection.
+    required: false
+    choices: ["disable", "certificate-inspection", "deep-inspection"]
+
+  ssl_unsupported_ssl:
+    description:
+      - Action based on the SSL encryption used being unsupported.
+      - choice | bypass | Bypass.
+      - choice | inspect | Inspect.
+      - choice | block | Block.
+    required: false
+    choices: ["bypass", "inspect", "block"]
+
+  ssl_untrusted_cert:
+    description:
+      - Allow, ignore, or block the untrusted SSL session server certificate.
+      - choice | allow | Allow the untrusted server certificate.
+      - choice | block | Block the connection when an untrusted server certificate is detected.
+      - choice | ignore | Always take the server certificate as trusted.
+    required: false
+    choices: ["allow", "block", "ignore"]
+
+  ssl_exempt:
+    description:
+      - EXPERTS ONLY! KNOWLEDGE OF FMGR JSON API IS REQUIRED!
```

Comment:
To be in consistent to Ansible modules, can you please change the description/format in single statements with formatting like: 
```suggestion
      - EXPERTS ONLY! KNOWLEDGE OF FMGR JSON API IS REQUIRED!
        List of multiple child objects to be added. Expects a list of dictionaries. 
        Dictionaries must use FortiManager API parameters, not the ansible ones listed below. 
        If submitted, all other prefixed sub-parameters ARE IGNORED.
```